bin/rspecq

```diff
@@ -65,6 +66,11 @@ OptionParser.new do |o|
     opts[:report_timeout] = v
   end
 
+  o.on("--max-requeues N", Integer, "Sets the number of max requeues to N " \
+       "times (default: #{DEFAULT_MAX_REQUEUES}).") do |v|
```

Comment:
We better keep this help text consistent with the one you used in the README (changed it a bit):

> Retry failed examples up to N times before consider them legit failures (default: 3).